README.md

```diff
@@ -51,6 +51,59 @@ zwadp -c iso --oid oid_m31.dat --oid fakes/oid_m31_fake.dat --feature feature_m3
 zwadp -h
 ```
 
+### Active anomaly detection
+
+Lets use the same data from [Zenodo](https://zenodo.org/record/4318700).
+```shell
+cd data
+zwad-zenodo
+```
+
+It is better to create temporary directory to run active anomaly detection:
+```shell
+mkdir ../tmp
+cd tmp
+```
+
+```shell
+# Run the algorithm with M31 dataset
+zwaad --oid ../data/oid_m31.dat --feature ../data/feature_m31.dat --feature-names ../data/feature_m31.name --budget 4
+```
+
+The script output may be as the following:
+```
+Check https://ztf.snad.space/dr4/view/695211400124577 for details
```

Comment:
Should we say that instead of this line browser can be open? It is what happened to me